lib/inspec/resources/opa_cli.rb

```diff
@@ -22,6 +28,10 @@ def allow
       @content["result"][0]["expressions"][0]["value"] if @content["result"][0]["expressions"][0]["text"].include?("allow")
     end
 
+    def resource_id
+      "#{@policy}:#{@query}"
```

Comment:
This might lead to nil error too, since in line 17 `@policy = opts[:policy] || nil`

Reply:
Same here there is validation.